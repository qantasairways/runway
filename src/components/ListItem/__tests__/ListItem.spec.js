import React from 'react';
import { shallow } from 'enzyme';
import { ListItem } from '..';

describe('ListItem', () => {
  it('should match the snapshot when a simple textual child is given', () => {
    const component = shallow(<ListItem>Economy</ListItem>);

    expect(component).toMatchSnapshot();
  });
});
