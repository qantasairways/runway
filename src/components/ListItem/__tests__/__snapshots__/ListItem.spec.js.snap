// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItem should match the snapshot when a simple textual child is given 1`] = `"Economy"`;
